gigasecond.md

```diff
@@ -1 +1 @@
-A gigasecond is one billion (10**9) seconds.
+A gigasecond is 10^9 (1,000,000,000) seconds.
```

Comment:
`^` does mean "exclusive or" in some programming languages, so this is ambigous as well. Since we have some of them in the list of tracks, I'd try to avoid that completely by either writing `1e9` (afaik every language has some similar way to write floats) or by using `'SUPERSCRIPT NINE' (U+2079)` (10⁹). But the latter might cause trouble on some misconfigured systems. There are still systems not properly prepared for unicode on the market :(